common/constants.go

```diff
@@ -297,3 +297,8 @@ const (
 	ContentTypeText = "text/plain"
 	ContentTypeXML  = "application/xml"
 )
+
+// Constants for Notification Category
+const (
+	NotificationCategoryDisconnectionAlert = "DISCONNECTION_ALERT"
```

Comment:
shorten the variable:
`DisconnectAlert = "Disconnection"`

Reply:
modified